openy.install

```diff
@@ -1002,3 +1002,12 @@ function openy_update_8068() {
       ->save(TRUE);
   }
 }
+
+/**
+ * Disable /node page.
+ */
+function openy_update_8069() {
```

Comment:
Please move this update to `modules/openy_features/openy_node/openy_node.install`

Reply:
I moved it.